action.yaml

```diff
@@ -88,6 +92,20 @@ runs:
             helm repo add --username ziltoidbot --password ${{ inputs.helm_pat }} remotecompany https://raw.githubusercontent.com/remotecompany/charts/main/index
         fi
 
+    - name: Validate Helm templates
+      shell: bash
+      run: |
+        helm secrets template mailersend helm/ --kube-version=1.19.12 --namespace "${{ inputs.validate_helm_env }}" -f helm/values."${{ inputs.validate_helm_env }}".yaml -f helm/secrets."${{ inputs.validate_helm_env }}".yaml > "${{ inputs.validate_helm_env }}".yaml 
+        /usr/local/bin/kubeval --schema-location https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master/ -v 1.19.12 --strict --ignore-missing-schemas --filename=""${{ inputs.validate_helm_env }}".yaml" "${{ inputs.validate_helm_env }}".yaml
```

Comment:
K8s version should be configurable on input so we can make sure we can validate on different versions